recipes/acpype/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "acpype" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+    url: https://github.com/alanwilter/{{ name }}/archive/d99be8e17e3f999e06a401e5efdbe5f2fc9a2959.tar.gz
+    sha256: 43736873feaa791c76ee02a6eae6aba5fb661149a915d8772ec5a4a1f920473d
+build:
+    number: 0
+    script: python recipe/setup_conda.py install --single-version-externally-managed --record=record.txt
```

Could we use `pip` instead:
```suggestion
      script:
        - mv recipe/setup_conda.py recipe/setup.py
        - {{ PYTHON }} -m pip install recipe/ -vv"
```
(it handles metadata in a more friendly way that using the `setup.py` directly)